attendance/models.py

```diff
@@ -20,7 +21,51 @@ class Project(models.Model):
     def __str__(self):
         """Just name of project."""
         return self.name
-
+        
+        
+class Contract(models.Model):
+    contract_type = models.CharField(
+        max_length=20,
+        null=False,
+        blank=False
+    )
+    hours_quota = models.IntegerField(
+        default=0,
+        validators=[
+            MaxValueValidator(8),
+            MinValueValidator(0)
+        ]
+     )
+    
+    def __str__(self):
+        """Just contract type."""
+        return self.contract_type
+        
+        
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    contracts = models.ManyToManyField(Contract)
+    
+    def get_hours_quota(self):
```

Comment:
This could be `property` [more info here](https://stackoverflow.com/questions/17330160/how-does-the-property-decorator-work)